src/test/java/com/code_intelligence/jazzer/mutation/mutator/proto/BuilderMutatorProto3Test.java

```diff
@@ -502,4 +506,63 @@ void testEmptyMessage3() {
     }
     assertThat(builder.build()).isEqualTo(EmptyMessage3.getDefaultInstance());
   }
+
+  @Test
+  void testAnyField3() throws InvalidProtocolBufferException {
```

Comment:
Add a test for missing `AnySource` annotation.

Reply:
Added one that just checks for no crashes as the general mutation behavior is covered by the generic message field mutator logic.